Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -1035,7 +1035,8 @@ class NewTabPageSetting: Setting {
     }
 
     override func onClick(_ navigationController: UINavigationController?) {
-        let viewController = NewTabChoiceViewController(prefs: profile.prefs)
+        let viewController = NewTabContentSettingsViewController()
+        viewController.profile = profile
```

Comment:
Can we pass `profile` as part of the constructor?

Reply:
This is actually the same pattern we use everywhere in settings. To be consistent I'll keep it like this